integration-tests/benchmark_test.go

```diff
@@ -37,6 +37,11 @@ func (b *BenchmarkTestSuiteBase) StartPerfTools() {
 	perf := ReadEnvVar("COLLECTOR_PERF_COMMAND")
 	bpftrace := ReadEnvVar("COLLECTOR_BPFTRACE_COMMAND")
 	bcc := ReadEnvVar("COLLECTOR_BCC_COMMAND")
+	image := "quay.io/rhacs-eng/collector-performance"
+
+	if runtime.GOARCH != "amd64" {
+		image = "quay.io/rhacs-eng/" + runtime.GOARCH + "/collector-performance"
```

Comment:
What about specifying this image as environment variables as well? This might be beneficial for self-built images or would further abstract arch specifics for now... I would guess that in future, this container image will be built w/ manifest-list / multi-arch support.

Reply:
arch specifics have already been removed from here. Cant see this in the "Files Changed" tab.